docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -2082,7 +2078,60 @@ Daml Script has been used for demonstrative purposes in this document, however
 usually the complexities of live upgrades comes with your workflows, not the data
 transformations themselves. You can use Daml Script (with Canton) to test some
 isolated simple cases, but for thorough tests of you system using SCU, you should
-prefer full workflow testing, as below.
+prefer full `Workflow Testing <#workflow-testing>`__.
+
+When testing your upgrades with Daml Script, we recommend you place your Daml Script tests
+in a separate package which depends on all versions of your business logic. This testing
+package should not be uploaded to the ledger if possible, as it cannot be dropped in an upgrade.
```

Comment:
"as it cannot be dropped in an upgrade": technically you can no? Since it's a separate package that presumably no other package depends on, you can choose not to upload any upgrade to it. You can even unvet it.

It seems to me the real reason we don't want users to upload such a package is because it depends on daml-script and we don't want users to upload daml-script-lts.

Reply:
This is true, but given unvetting isn't deletion, they would still be stuck with it on their ledger.
I'll try a better wording, but i really do mean in general, dont upload daml script

Reply:
Oh absolutely! My point was maybe what we should say just that: don't upload that package because it will upload daml-script and we don't want that.